Website/src/rendering/src/components/Vendors/VendorsGrid.tsx

```diff
@@ -10,8 +16,15 @@ type VendorsGridProps = ComponentProps & {
 };
 
 const VendorsGrid = (props: VendorsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasVendors = props.fields;
```

Comment:
`hasVendors` is an object with that code. We could use `!!props.fields` for it to be a boolean and consume less memory.